app/views/tournament/form.scala

```diff
@@ -281,3 +300,13 @@ final private class TourFields(form: Form[?], tour: Option[Tournament])(using Pa
         a(cls := "show")(trans.showAdvancedSettings())
       )
     )
+
+private opaque type FormPrefix = Option[String]
+object FormPrefix extends TotalWrapper[FormPrefix, Option[String]]:
+  val empty           = FormPrefix(none)
+  def make(s: String) = FormPrefix(s.some)
+
+extension (f: Form[?])
+  def prefix(name: String)(using prefixOpt: FormPrefix) = f(
+    prefixOpt.fold(name)(prefix => s"$prefix.$name")
+  )
```

Comment:
This is the way I found to allow nested form with minimal hassle and code change, but maybe there's a better way.